### ISSUE TITLE
[URGENT] Classes not generating

### ISSUE THREAD

**Issue by raghavsatyadev** [simolus3/moor#2418]
**Problems in generating files**

I am getting this error from the build command 

```shell
dart run build_runner build --delete-conflicting-outputs > build_output.txt
```

```
[INFO] Generating build script...
[INFO] Generating build script completed, took 586ms

[INFO] Precompiling build script......
[WARNING] /Pub/Cache/hosted/pub.dev/drift_dev-2.7.0/lib/src/analysis/results/dart.dart:249:7: Error: The non-abstract class '_AddFromDartType' is missing implementations for these members:
 - TypeVisitor.visitInvalidType
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class _AddFromDartType extends TypeVisitor<void> {
      ^^^^^^^^^^^^^^^^
/Pub/Cache/hosted/pub.dev/analyzer-5.12.0/lib/dart/element/type_visitor.dart:19:5: Context: 'TypeVisitor.visitInvalidType' is defined here.
  R visitInvalidType(InvalidType type);
    ^^^^^^^^^^^^^^^^
[INFO] Precompiling build script... completed, took 10.8s

[SEVERE] Failed to precompile build script .dart_tool/build/entrypoint/build.dart.
This is likely caused by a misconfigured builder definition.
```

**Flutter --version output:**

```
Flutter 3.11.0-2.0.pre.83 • channel master • https://github.com/flutter/flutter.git
Framework • revision 8ab782d08a (7 hours ago) • 2023-05-09 17:50:11 -0700          
Engine • revision 8ca16cba8c                                                       
Tools • Dart 3.1.0 (build 3.1.0-89.0.dev) • DevTools 2.23.1  
```

pubspec.yaml required packages:

```
dependencies::
    drift: ^2.7.0
    drift_local_storage_inspector: ^0.5.0
    path: ^1.8.3
    path_provider: ^2.0.15
    sqlite3_flutter_libs: ^0.5.13

dev_dependencies:
    build_runner: ^2.3.3
    drift_dev: ^2.7.0
    source_gen: ^1.3.1
```

**Comment by simolus3** [comment]
Duplicate of https://github.com/simolus3/drift/issues/2419.

You can temporarily fix this by adding this to your pubspec:

```yaml
dependency_overrides:
  analyzer: '>=5.10.0 <5.12.0'
```

But I will release a compatible `drift_dev` version in next hours after doing some testing.

**Comment by simolus3** [comment]
Fixed in `drift_dev: 2.8.0`.
